src/status_im/utils/priority_map.cljs

```diff
@@ -0,0 +1,229 @@
+(ns status-im.utils.priority-map
```

Comment:
@yenda is it copied from somewhere or that's yours implementation?

Reply:
yes it's https://github.com/tailrecursion/cljs-priority-map/blob/master/src/cljs/tailrecursion/priority_map.cljs with a bug fixv for keys and val fns

Reply:
do you think I should clone the repo instead @rasom ?